src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -82,6 +77,16 @@ public UniquePersonList(UniquePersonList source) {
     }
 
 
+    /**
+     * Returns an unmodifiable java List view with elements cast as immutable {@link ReadOnlyPerson}s.
+     * Sorted by name.
+     */
+    public List<ReadOnlyPerson> sortedImmutableListView() {
+        internalList.sort(Comparator.comparing(i -> i.getName().toString()));
```

Comment:
How about implementing such sorting in OOP fashion? This method is more like FP.